frontend/src/mocks/msw/handlers/auth.ts

```diff
@@ -0,0 +1,71 @@
+import { rest } from 'msw'
```

Comment:
is there a way to look at the behaviour of components in Storybook for different API responses? e.g. change whether the call to `/otp/generate` succeeds or fails, and see how the component responds to each?

Reply:
yea, it is possible. We just need to create alternative handlers for the error cases. I'll add one non-whitelisted test case then.

Reply:
added in 9363659